### PR TITLE
Create $ruslana.json

### PR DESCRIPTION
**Pull request by rh-pr**
jscraftcamp_ruslana

I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] I read the `THE IMPORTANT STUFF` at [https://jscraftcamp.org/registration](https://jscraftcamp.org/registration), the JSON file follows the [template](https://github.com/jscraftcamp/website/blob/main/participants/_template.json), and contains the mandatory fields like `realName`, `githubAccountName`, `when`, `iCanTakeNotesDuringSessions`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`.
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already
- [x] I asked my company about sponsoring (see open items at: https://github.com/orgs/jscraftcamp/projects/7)

Are you from a sponsoring company?

- [ ] Yes, I am from company ?????

### CODE REVIEW

**Comment by rh-pr** [comment]
first comment

**Review by rh-pr — COMMENTED** [review]
changed the answer

File: participants/$ruslana.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	"when": {
+		// June 7th, 2024
+		"friday": true,
+		// June 8th, 2024
+		"saturday": true
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": true,
+	// your current interests (JS and in general) (required)
+	"tags": ["ReScript", "Svelte", "Zustand", "GraphQL"],
```
Comment (Narigo) [inline participants/$ruslana.json:28]: Are you sure these are the tags you're interested in? 😅 Looks like the ones we put in the template 🙂

**Comment by Narigo** [comment]
Hi @rh-pr, like everybody else who registered only recently, we need to put you on the waiting list for now

**Comment by Narigo** [comment]
Welcome to the JSCC24 @rh-pr ! You are registered for both days! 🥳

Would be great if you could update the tags - we merged your PR though so you'll be on the list for both days that we need to provide for the location. Looking forward to seeing you there and happy to learn what interactive capabilities you create with JS 🙂

**Pull request merged**